res/enkf/ensemble_config.py

```diff
@@ -255,10 +252,10 @@ def add_field(self, key, eclipse_grid: EclGrid) -> EnkfConfigNode:
         """@rtype: EnkfConfigNode"""
         return self._add_field(key, eclipse_grid).setParent(self)
 
-    def getKeylistFromVarType(self, var_mask: EnkfVarType) -> StringList:
+    def getKeylistFromVarType(self, var_mask: EnkfVarType) -> list:
```

Comment:
Can we use `list` as return-type, or does it have to be `List` from the typing-package?

Reply:
`list` is fine, but `List` is probably better as I can do `List[str]` then, will change.